contrib/for-tests/mount-tester/mt.go

```diff
@@ -152,3 +160,41 @@ func readWriteNewFile(path string) error {
 
 	return readFileContent(path)
 }
+
+func readFileContentInLoop(path string) error {
+	if path == "" {
+		return nil
+	}
+	var content []byte
+	//Expected content "mount-tester new file\n", length (22), 
+	//retry during 240 second
+	content = testFileContent(path, 22, 240)
```

Comment:
@dalanlan I think 240 needs to be a CLI param to the mount tester, and '22' ought to be a constant